addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/Config.java

```diff
@@ -37,13 +38,16 @@
     public static final String COMETVISU_ICON_ENABLE_MAPPING_PROPERTY = "enableMapping";
 
     public static final String COMETVISU_WEBFOLDER_PROPERTY = "webFolder";
+    public static final String COMETVISU_AUTODOWNLOAD_PROPERTY = "autoDownload";
 
-    public static String COMETVISU_WEBFOLDER = "runtime/web/";
+    public static String COMETVISU_WEBFOLDER = "/var/www/cometvisu";
```

Comment:
A hard coded absolute address definitely looks baaaad :-)
If this is the folder where the runtime automatically downloads stuff to, I would agree it should be the userdata folder (as this is the only one officially writable). To get the location from code `ConfigConstants.getUserDataFolder()` should be used.